.config/dotnet-tools.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Please delete this file. Why do you have it here? Did you compile and run project using some console commands instead of visual studio?